csv_parser/csv_parser.py

```diff
@@ -123,6 +128,15 @@ def is_custom_content():
     if perimeter_filter:
         df = df[pd.notna(df[perimeter_filter])]
 
+    # Replace 'Cardinalité' column values with the relevant perimeter column values (whenever the value is not 'X')
+    if perimeter_filter:
+        df['Cardinalité'] = df.where(df[perimeter_filter] == 'X', df[perimeter_filter], axis=0)['Cardinalité']
+
+    # Deleting perimeter columns. N.B: dropping a
+    # column (obviously) reduces the length df.columns, so we iterate in reverse order
+    for i in reversed(range(0, len(params['perimeterColumns']))):
+            df.drop(df.columns[params['perimeterColumns'][i]], axis=1, inplace=True)
```

Comment:
```suggestion
    for i in sorted(params['perimeterColumns'], reverse=True):
            df.drop(df.columns[i], axis=1, inplace=True)
```

Reply:
Ca me semble plus clean et plus safe au cas où perimetersColumns n'était initialement pas trié (pas le cas avec ton code mais on sait jamais, down the line...)